jena-fuseki2/jena-fuseki-ui/tests/unit/components/serverstatus.vue.spec.js

```diff
@@ -33,7 +33,6 @@ describe('ServerStatus', () => {
         mocks: {
           $fusekiService: {
             async getServerStatus () {
-              console.log('Hola!')
```

Comment:
Sorry! I had noticed these console log lines, but I thought I had intentionally written that somewhere... but searching it revealed I had simply forgotten to remove this debug line :bow: